shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -10,165 +10,112 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.redhat.service.smartevents.infra.api.APIConstants;
-import com.redhat.service.smartevents.infra.exceptions.definitions.platform.HTTPResponseException;
 import com.redhat.service.smartevents.infra.models.dto.BridgeDTO;
 import com.redhat.service.smartevents.infra.models.dto.ManagedResourceStatus;
 import com.redhat.service.smartevents.infra.models.dto.ProcessorDTO;
-import com.redhat.service.smartevents.shard.operator.exceptions.DeserializationException;
-import com.redhat.service.smartevents.shard.operator.metrics.ManagerRequestStatus;
-import com.redhat.service.smartevents.shard.operator.metrics.ManagerRequestType;
-import com.redhat.service.smartevents.shard.operator.metrics.MetricsService;
-import com.redhat.service.smartevents.shard.operator.utils.WebClientUtils;
 
 import io.quarkus.scheduler.Scheduled;
 import io.smallrye.mutiny.Uni;
-import io.vertx.mutiny.core.buffer.Buffer;
-import io.vertx.mutiny.ext.web.client.HttpRequest;
-import io.vertx.mutiny.ext.web.client.HttpResponse;
-import io.vertx.mutiny.ext.web.client.WebClient;
 
 @ApplicationScoped
 public class ManagerSyncServiceImpl implements ManagerSyncService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ManagerSyncServiceImpl.class);
 
-    @Inject
-    ObjectMapper mapper;
+    private final BridgeHandler BRIDGE_HANDLER = new BridgeHandler();
+
+    private final ProcessorHandler PROCESSOR_HANDLER = new ProcessorHandler();
 
     @Inject
-    WebClient webClientManager;
+    ManagerClient managerClient;
 
     @Inject
     BridgeIngressService bridgeIngressService;
 
     @Inject
     BridgeExecutorService bridgeExecutorService;
 
-    @Inject
-    EventBridgeOidcClient eventBridgeOidcClient;
-
-    @Inject
-    MetricsService metricsService;
-
+    @Override
     @Scheduled(every = "30s")
-    void syncUpdatesFromManager() {
+    public void syncUpdatesFromManager() {
         LOGGER.debug("Fetching updates from Manager for Bridges and Processors to deploy and delete");
-        fetchAndProcessBridgesToDeployOrDelete().subscribe().with(
+        doBridgeActions().subscribe().with(
```

Comment:
I moved invocation to `protected doXXXActions()` as there are separate tests for `Bridge`'s and `Processor`'s.

Reply:
I can then more easily invoke only processing for `Bridge`'s or `Processor`'s.